bin/functions.sh

```diff
@@ -1,3 +1,11 @@
+#!/bin/sh
+
+convert () {
+  ruby -ryaml -rjson -e 'puts JSON.parse(ARGF.read).to_yaml' < __output__/admin-schema-processed-$VERSION.json > __output__/admin-schema-redocly-$VERSION.json
+  ruby -ryaml -rjson -e 'puts JSON.parse(ARGF.read).to_yaml' < __output__/customer-schema-processed-$VERSION.json > __output__/admin-schema-redocly-$VERSION.json
+  ruby -ryaml -rjson -e 'puts JSON.parse(ARGF.read).to_yaml' < __output__/guest-schema-processed-$VERSION.json > __output__/admin-schema-redocly-$VERSION.json
+}
```

Comment:
Need to update the output names for Customer and Guest.

Reply:
Good catch. It's a copy/paste bug.